mteb/overview.py

```diff
@@ -278,6 +290,7 @@ def get_tasks(
     exclusive_language_filter: bool = False,
     modalities: list[MODALITIES] | None = None,
     exclusive_modality_filter: bool = False,
+    exclude_aggregate: bool = False,
```

Comment:
Wanted true by default
```suggestion
    exclude_aggregate: bool = True,
```

Reply:
Done! This may change the behavior of get_tasks for existing callers, as aggregate tasks will no longer be returned by default.

Reply:
You're right. I misread the issue comment - we do want default to include all tasks.

Reply:
Let's revert back to False by default then. Thanks!

Reply:
Reverted!